### PR TITLE
Project Addition: Added a chrome extension project

### PR DESCRIPTION
**Pull request by abhinav-m22**
## Related Issue

* Added a project for the issue - #558

### CODE REVIEW

**Comment by priyankarpal** [comment]
You have already created a PR before & also got merged you don't need to add full code. add this code to the previous code to add multiple projects. 
```
      {
        "link": "https://github.com/abhinav-m22/google-keep-clone-2.0",
        "title": "Google Keep Clone 2.0 - Notes App",
        "description": "An updated version of Google Keep Clone with updated UI and UX. It 
         can add, remove and archive notes and is styled using MaterialUI, CSS and JSX.",
        "tech": ["ReactJS", "Material UI", "JavaScript", "CSS", "JSX"]
      }
```

**Comment by abhinav-m22** [comment]
I've done the required changes... Please check now

File: src/DB/projects.json [hunk 1]
```diff
@@ -973,6 +973,18 @@
...
+        "title": "YouTube Bookmark Chrome Extension",
+        "description": "The project is a Google Chrome extension for adding Bookmarks of YouTube video timestamps. Users can bookmark a video at particular timestamp and resume watching it later.",
+        "tech": ["JavaScript", "DOM", "HTML", "CSS"]
+      },
+      {
+        "link": "https://github.com/abhinav-m22/yt-bookmark-extension",
+        "title": "YouTube Bookmark Chrome Extension",
+        "description": "The project is a Google Chrome extension for adding Bookmarks of YouTube video timestamps. Users can bookmark a video at particular timestamp and resume watching it later.",
+        "tech": ["JavaScript", "DOM", "HTML", "CSS"]
       }
```
Comment (priyankarpal) [inline src/DB/projects.json:988]: you have repeated the code

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by abhinav-m22** [comment]
Sorry for the issues. I made the mistake in a hurry. Please check now

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**